app/controllers/show_controller.rb

```diff
@@ -0,0 +1,5 @@
+class ShowController < ApplicationController
```

Comment:
this should rather be a method in the posts controller, and not it's own controller.